.github/workflows/ci.yml

```diff
@@ -77,13 +77,13 @@ jobs:
         if: |
           !startsWith(matrix.os, 'windows')
         run: |-
-          curl -fsSL https://deno.land/x/install/install.sh | sh -s v1.5.1
+          curl -fsSL https://deno.land/x/install/install.sh | sh -s v1.7.2
```

Comment:
Why did you update these?

Reply:
> Why did you update these?

This is Discard talk about it
![](https://user-images.githubusercontent.com/13204332/107304089-5c9aa780-6abb-11eb-9db6-754ad8374705.png)